source/services/date/date.service.ts

```diff
@@ -34,10 +34,12 @@ export interface IDateUtility {
 	formatDate(date: string | Date | moment.Moment, dateFormat?: string): string;
 	sameDate(date1: string | Date | moment.Moment, date2: string | Date | moment.Moment, date1Format?: string, date2Format?: string): boolean;
 	sameDateTime(date1: string | Date | moment.Moment, date2: string | Date | moment.Moment, date1Format?: string, date2Format?: string): boolean;
+	setOffset(millis: number);
 }
 
 export class DateUtility {
 	private baseFormat: string = defaultFormats.isoFormat;
+	private offsetFromServer: number;
```

Comment:
Do we want to abstract this to just `offset`, rather than explicitly saying 'from server' here? In theory, this could be used for any reason an app developer would have to offset the current time.